packages/squiggle-lang/__tests__/Reducer/Reducer_mapReduce_test.res

```diff
@@ -9,3 +9,8 @@ describe("map reduce", () => {
   testEvalToBe("arr=[1,2,3]; reverse(arr)", "Ok([3,2,1])")
   testEvalToBe("check(x)=(x==2);arr=[1,2,3]; keep(arr,check)", "Ok([2])")
 })
+
+Skip.describe("map reduce (sam)", () => {
+  testEvalToBe("addone(x)=x+1; map(2, addone)", "Error???")
+  testEvalToBe("addone(x)=x+1; map(2, {x: addone})", "Error???")
+})
```

Comment:
why are these skipped?

Reply:
Those are weirdness in Error messages.  The error messages need to be more helpful. However, I cannot fix the error messages before the new parser. So we are skipping the issues in error messages.

Those are all expressions that would not evaluate.  And they don't evaluate. However Sam reminded  me to fix the error messages in the future